srt-protocol/src/protocol/pending_connection/connect.rs

```diff
@@ -219,7 +223,7 @@ mod test {
             }),
         });
 
-        let resp = c.handle_packet((first, test_remote()), Instant::now());
+        let resp = c.handle_packet((0, first, test_remote()), Instant::now());
```

Comment:
These zeros are messy--another reason to go with `wire_size` in #134

Reply:
yes, ok, I'll remove